decidim-assemblies/spec/permissions/decidim/assemblies/permissions_spec.rb

```diff
@@ -280,7 +280,7 @@
     it_behaves_like(
       "access for roles",
       org_admin: true,
-      admin: false,
+      admin: true,
```

Comment:
If it is already `org_admin`, why change the `admin` role to to true? Shouldn't we keep this test as is and add another one to check when `org_admin: false` and `admin: true`?

Reply:
This test checks what type of role can create assemblies, now the process (or assembly) admin can also create them, I don't understand your question.

Reply:
I think the "access for roles" naming could be a bit clear, but I agree with @LauraJaimeC, that it isn't need to add the explicit check, as this is a shared example (defined at line 31) and in this example group.